testsys: Add support for k8s workloads (#2830)
bottlerocket-os/bottlerocket: TESTING.md

To see the state of the tests as they run use `cargo make watch-test`.

### Testing Workloads

> Workload tests are tests designed to run on a pod in a cluster.

Reviewer: We can do ECS workload tests as well, right?

```suggestion
Workload tests are tests designed to run as an orchestrated container.
```

Author: We can't do ECS yet. That will come in the next TestSys release.
